national-insurance.conf

```diff
@@ -6125,7 +6127,7 @@
       fishermen = £4.10
       volunteer = £6.15
     }
-    lel = £11908
+    lel = £12570
```

Comment:
Was this being used ?
AC suggests we remove it but theres nothing in the ticket to confirm if it was being used

Reply:
Reinstated